.github/workflows/test.yml

```diff
@@ -239,7 +239,7 @@ jobs:
   #   timeout-minutes: 10
   #   steps:
   #     - uses: actions/checkout@v3
-  #     - uses: actions/setup-go@v3
+  #     - uses: actions/setup-go@v4
```

Comment:
kinda interesting that depandabot finds and updates versions that are commented. This makes me think that dependabot does a find + replace rather than using metadata on what deps are actually running in CI or go.mod